python/tests/conftest.py

```diff
@@ -5,15 +5,16 @@
 import pytest
 
 import semantic_kernel as sk
+from semantic_kernel.settings import KernelSettings, OpenAISettings, load_settings
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture()
 def create_kernel():
     kernel = sk.Kernel()
     return kernel
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture()
```

Comment:
Yet another thing I forgot to mention. Apologies. I wanted to ask you about this, and made this change so that I don't forget it.

Did you intend to use `scope="session"` in the fixture? The default scope is `"function"` which basically means that the fixture gets recreated for every test. Other fixture scopes, in my experience, cause issues down the line, once the number of tests increases. Since non-function scope fixtures are shared across multiple tests, if any one of those tests makes a change to the result returned by the fixture, downstream tests see the changed value. And since the order in which the tests are run is not fixed (especially across code changes), it sometimes leads to failures which are not easily reproducible either.

Fixtures with larger scopes are typically used when the object being constructed in the fixture is heavy and takes time to be created or if you need the shared property of the fixture. These fixtures don't seem to be satisfying either of these criteria, so I just wanted to check with you on them.